package/win32/make-package.bat

```diff
@@ -12,6 +12,11 @@ REM Remove Rtools from path
 CALL SET PATH=%PATH:C:\Rtools\bin=%
 CALL SET PATH=%PATH:C:\Rtools\gcc-4.6.3\bin=%
 
+REM Remove Git from PATH (otherwise cmake complains about
+REM finding 'sh.exe')
+CALL set PATH=%PATH:C:\Program Files (x86)\Git\bin=%
```

Comment:
This shouldn't affect the build servers; it just helps when performing a local package build (removes Git from PATH)